src/main/scala/nl.knaw.dans.easy.solr4files/components/Bag.scala

```diff
@@ -48,7 +48,7 @@ case class Bag(storeName: String,
     // gov.loc.repository.bagit.reader.ManifestReader reads files, we need URL or stream
     for {
       url <- vault.fileURL(storeName, bagId, "manifest-sha1.txt")
-      lines <- url.readLines
+      lines <- url.readLines()
```

Comment:
Does `readLines` alter global state? The convention in Scala is to call purely functional methods without parentheses. See: https://docs.scala-lang.org/style/method-invocation.html#arity-0

Reply:
@janvanmansum it's necessary here, since `readLines` now takes default parameters... Not sure if it's the way to go, by these parentheses have nothing to do with global state in this instance.